<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QTextEdit" name="viewTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="unlockBtn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>550</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>解锁</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restorationBtn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>550</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>5</y>
     <width>291</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>串口号：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="serialPortNameComb">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>170</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>数据位：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="dataBitsComb">
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>波特率：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="baudRateComb">
            <item>
             <property name="text">
              <string>1200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>115200</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>停止位：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="stopBitComb">
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>校验位：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="parityBitComb">
            <item>
             <property name="text">
              <string>0</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QPushButton" name="searchSerialBtn">
            <property name="text">
             <string>检测串口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openSerialBtn">
            <property name="text">
             <string>打开串口</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>激光功率：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="openLaserLineEdit">
          <property name="text">
           <string>800</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openLaserBtn">
          <property name="text">
           <string>打开激光</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QPushButton" name="setTheOriginBtn">
          <property name="text">
           <string>设置原点</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="RegressionOriginBtn">
          <property name="text">
           <string>回归原点</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <widget class="QPushButton" name="previewBtn">
        <property name="text">
         <string>预览</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="modifyGrblBtn">
        <property name="text">
         <string>修改grbl</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QPushButton" name="pauseBtn">
        <property name="text">
         <string>暂停</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="continueBtn">
        <property name="text">
         <string>继续</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>F速度：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fSpeedLineEdit">
            <property name="text">
             <string>800</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>X轴步距：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="xStepLineEidt">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Y轴步距：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="yStepLineEdit">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Z轴步距：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="zStepLineEdit">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPushButton" name="xLeftMoveBtn">
            <property name="text">
             <string>X-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="xRightMoveBtn">
            <property name="text">
             <string>X+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="xReverseBtn">
            <property name="text">
             <string>X轴反向</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="yLeftMoveBtn">
            <property name="text">
             <string>Y-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="yRightMoveBtn">
            <property name="text">
             <string>Y+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="yReverseBtn">
            <property name="text">
             <string>Y轴反向</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QPushButton" name="zLeftMoveBtn">
            <property name="text">
             <string>Z-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="zRightMoveBtn">
            <property name="text">
             <string>Z+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="zReverseBtn">
            <property name="text">
             <string>Z轴反向</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
